site/_data/master-toc.yml

```diff
@@ -73,6 +73,8 @@ toc:
         url: /code-standards
       - page: Website guidelines
         url: /website-guidelines
+      - page: Style guide
```

Comment:
suggest calling this `Documentation Style Guide`

Reply:
+1

Reply:
updated to Documentation style guide in the navigation